news/2023/2023-04-23-new-featured-artist-7-7.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "New Featured Artist: 7\_7"
+date: 2023-04-23 07:00:00 +0000
+---
+
+Anarchy enters our Featured Artist library through top-tier osu!taiko player **7\_7**'s chaotic sound.
+
+![](https://assets.ppy.sh/artists/356/header.jpg)
+
+Before even touching music production, [**7\_7**](https://osu.ppy.sh/users/7491106) was one of us.
+
+Peaking at **global rank #5** in osu!taiko, **7\_7** has encountered and endless slew of insane songs through his drum bashing journey, leading him to produce a highly specific style of music that's perfect for rhythm games: *ultra-dense breakcore tracks with intensity that no other songs dare to match.*
+
+**25** of those songs are ready for mapping from [**7\_7**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/356), including the [osu!original](/wiki/osu!_originals) *7\_7 feat. kasane vavzed - 7D* that debuted in the Grand Finals pool of [TWC 2023](/wiki/Tournaments/TWC/2023) (which happened a few minutes ago, by the way).
+
+If you're struggling to map something by **7\_7** (or any of our other Featured Artists), check out the `#featured-artists` channel in the [osu!dev Discord Server](https://discord.gg/ppy).
```

Comment:
```suggestion
If you're struggling to map something by **7\_7** (or any of our other Featured Artists), check out the `#featured-artists` channel in the [osu! Discord Server](https://discord.gg/ppy).
```

`dev` should be removed as to stay consistent with the name change + [wiki article changes.](https://osu.ppy.sh/wiki/en/Community/osu%21_Discord_server).